tests/cli/workflow/mip/test_cli_mip_base.py

```diff
@@ -46,7 +46,7 @@ def test_spring_decompression_needed_and_started(
     CompressAPI.decompress_spring.return_value = True
 
     # WHEN an MIP analysis is started
-    result = cli_runner.invoke(start_available, obj=dna_mip_context)
+    result = cli_runner.invoke(start_available, obj=mip_dna_context)
 
     # THEN command should run without errors
     assert result.exit_code == 0
```

Comment:
```suggestion
    assert result.exit_code == EXIT_SuCCESS
```
More readable. There is a constant you can import and reuse